tests/Custom/SelectTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
```

Comment:
@aik099 did you mean Behat\Mink\Tests\Driver\Form\SelectTest ? This test is more situated for  Selenium2Driver class, because in Mink it test set value by NodeElement. However the problem is more deeper, because we have two way of setting value, it's confusing.